docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -218,6 +218,48 @@ To remove a traffic light from `TrafficLightPublishPanel`, click the `RESET` but
 
 [Reference video tutorials](https://drive.google.com/file/d/1bs_dX1JJ76qHk-SGvS6YF9gmekkN8fz7/view?usp=sharing)
 
+### Driving across a crosswalk
+
+When driving across a crosswalk, ego vehicle will respond to objects on the crosswalk and the traffic light.
+
+#### Unsignalized crosswalk
+
+To pass through an unsignalized crosswalk, the ego vehicle will first slow down and stop for a safety check. Once safety is confirmed, the vehicle will proceed.
+
+1. Set an initial pose and a goal pose of the ego vehicle driving across an unsignalized crosswalk. A path will be planned.
+
+   ![unsignalized-placing-ego-vehicle](images/planning/passing-crosswalk/unsignalized-placing-ego-vehicle.png)
+
+2. Place a dummy pedestrian on the crosswalk and set its pose as if it is crossing the street.
+
+   ![unsignalized-crosswalk-start](images/planning/passing-crosswalk/unsignalized-crosswalk-start.png)
+
+3. Engage the ego vehicle. The ego vehicle will decelerate and stop before the crosswalk. The marked **crosswalk** in the figure indicates that the stopping behavior is casued by the objects on the **crosswalk**.
+
+   ![unsignalized-crosswalk-stop](images/planning/passing-crosswalk/unsignalized-crosswalk-stop.png)
+
+4. Wait for about three seconds. The ego vehicle will start moving again and cross the crosswalk.
```

Comment:
```suggestion
4. Wait for about a few seconds. The ego vehicle will start moving again and cross the crosswalk.
```

Actually, the time to wait for the pedestrian depends the project requirement. It is better to blur the time.